iam/x509-identity-mgmt/README.md

```diff
@@ -1121,6 +1121,7 @@ fore mentioned convention.
 | certificate.validity | integer | `365` | positive values | X509IDMGMT_CERTIFICATE_VALIDITY | certificate validity (in days). |
 | certificate.check.publickey | boolean | `true` | `true` or `false` | X509IDMGMT_CERTIFICATE_CHECK_PUBLICKEY | Flag indicating whether the CSR public key validation should be performed or not. |
 | certificate.check.subjectdn | boolean | `false` | `true` or `false` | X509IDMGMT_CERTIFICATE_CHECK_SUBJECTDN | Flag indicating whether the CSR SubjectDN field validation should be performed or not. |
+| certificate.belongsto.application | string[] | `["kafka-consumer", "vernemq", "v2k-bridge", "k2v-bridge"]` | String array, where each element must pass through the regular expression: `^[0-9a-zA-Z-]{1,64}$` | X509IDMGMT_CERTIFICATE_BELONGSTO_APPLICATION | list of application names to which the service can issue a certificate. |
```

Comment:
The value in the configuration may not even pass through the regular expression, but when requesting a certificate for the service, the regular expression `^[0-9a-zA-Z-]{1,64}$` will be applied to the payload _before_ checking if the application name is valid. That is, even setting a value that does not respect the regex, at the time of the request, it will be barred before the match with the configuration values.